examples/chef/common/chef-fan-control-manager.cpp

```diff
@@ -245,11 +234,8 @@ void ChefFanControlManager::FanModeWriteCallback(FanControl::FanModeEnum aNewFan
     switch (aNewFanMode)
     {
     case FanControl::FanModeEnum::kOff: {
-        if (mSpeedCurrent != 0)
```

Comment:
Waiting for https://github.com/project-chip/connectedhomeip/pull/36515 to land first